### ISSUE TITLE
[Bug] Metldown never occurred on 2.6.18

### ISSUE THREAD

**Issue by JohnMetroid** [LoafOrc/FacilityMeltdown#52]
**I HAVE COMPLETED THE FOLLOWING**
 - [x] I have already joined the [lethal company modding discord](https://discord.gg/lcmod) and gone to the [FacilityMeltdown thread](https://discord.com/channels/1168655651455639582/1191847266777042985) and checked the issue is not already present
 - [x] I have checked the issue page for both open and closed issues that could be relevant.
 - [x] I have attached my latest log file while having debug logs enabled.
 - [x] I have reproduced the issue.
 - [x] I will fill out the information below to the best of my ability.

## Describe the bug
Pulling apparatus causes meltdown effects, but meltdown never occurs

## To Reproduce
Steps to reproduce the behaviour:

Pull apparatus.  Wait more than 2 minutes.

## Expected behaviour
Meltdown occurs after 2 minutes

## Environment
 - OS: Windows: win 10
 - Lethal Company Version: v62 

## Modlist
 - FacilityMeltdown version: 2.6.18
 - LethalLib version: 0.16.1
 - TerminalAPI version: 1.5.5

01918031-8ba2-5efe-d14f-696c416e6089

### Additonal Context

```
[16:54:11.1323584] [Info   :me.loaforc.facilitymeltdown] Beginning Meltdown Sequence! I'd run if I were you!
[16:54:11.1363595] [Error  : Unity Log] ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection.
Parameter name: index
Stack trace:
System.Collections.Generic.List`1[T].get_Item (System.Int32 index) (at <787acc3c9a4c471ba7d971300105af24>:IL_0009)
FacilityMeltdown.MeltdownSequence.Behaviours.MeltdownHandler.SpawnEnemies () (at ./MeltdownSequence/Behaviours/MeltdownHandler.cs:198)
FacilityMeltdown.MeltdownSequence.Behaviours.MeltdownHandler.StartMeltdownClientRpc () (at ./MeltdownSequence/Behaviours/MeltdownHandler.cs:149)
FacilityMeltdown.MeltdownSequence.Behaviours.MeltdownHandler.__rpc_handler_3631067672 (Unity.Netcode.NetworkBehaviour target, Unity.Netcode.FastBufferReader reader, Unity.Netcode.__RpcParams rpcParams) (at <c2014084d570456095fee1b0549988cf>:IL_002E)
Unity.Netcode.RpcMessageHelpers.Handle (Unity.Netcode.NetworkContext& context, Unity.Netcode.RpcMetadata& metadata, Unity.Netcode.FastBufferReader& payload, Unity.Netcode.__RpcParams& rpcParams) (at <895801699cfc4b4ab52267f31e2a4998>:IL_004E)
Rethrow as Exception: Unhandled RPC exception!
UnityEngine.Logger:LogException(Exception, Object)
AsyncLoggers.Wrappers.Unity.<>c__DisplayClass4_0:<LogException>b__0() (at ./src/Wrappers/Unity/AsyncLoggerWrapper.cs:36)
AsyncLoggers.Wrappers.Unity.<>c__DisplayClass28_0:<wrapCallback>b__0() (at ./src/Wrappers/Unity/AsyncLoggerWrapper.cs:136)
AsyncLoggers.Wrappers.JobWrapper:LogWorker() (at ./src/Wrappers/JobWrapper.cs:84)
AsyncLoggers.Wrappers.LogJob:Execute() (at ./src/Wrappers/JobWrapper.cs:68)
Unity.Jobs.JobStruct`1:Execute(LogJob&, IntPtr, IntPtr, JobRanges&, Int32)

[16:54:11.1363595] [Info   : Unity Log] Lung apparatice was grabbed. Is owner: True
[16:54:12.1915373] [Info   : Unity Log] Flickering lights
[16:54:14.6921197] [Info   : Unity Log] Calling power switch off event from roundmanager
[16:54:15.9262953] [Info   :me.loaforc.facilitymeltdown] Newtonsoft.Json.Linq.JArray
```

[LogOutput.log](https://github.com/user-attachments/files/16731733/LogOutput.log)

**Comment by LoafOrc** [comment]
this was fixed in 2.6.19
also never is a bit of a stretch lmao considering how (at least in vanilla) the conditions were fairly rare

**Comment by JohnMetroid** [comment]
Thanks for the quick fix

It wasn't my intent to offend you, simply to report that despite waiting until midnight when the ship automatically left, the meltdown never triggered.

Cheers
